db/system_distributed_keyspace.cc

```diff
@@ -117,9 +118,10 @@ bool system_distributed_keyspace::is_extra_durable(const sstring& cf_name) {
     return cf_name == CDC_TOPOLOGY_DESCRIPTION;
 }
 
-system_distributed_keyspace::system_distributed_keyspace(cql3::query_processor& qp, service::migration_manager& mm)
+system_distributed_keyspace::system_distributed_keyspace(cql3::query_processor& qp, service::migration_manager& mm, service::storage_proxy& sp)
         : _qp(qp)
```

Comment:
Is query_processor still needed when we're switching to lower level api?

Reply:
Not all functions switch to the lower level api

Reply:
Ok.